src/main/java/io/fabric8/maven/docker/util/AnsiLogger.java

```diff
@@ -224,8 +219,8 @@ private void print(String txt) {
 
     private String colored(String message, Ansi.Color color, boolean addPrefix, Object ... params) {
         Ansi ansi = ansi().fg(color);
-        String msgToPrint = addPrefix ? prefix + message : message;
-        return ansi.a(format(evaluateEmphasis(msgToPrint, color), params)).reset().toString();
+        message = ansi.a(format(evaluateEmphasis(message, color), params)).reset().toString();
+        return addPrefix ? prefix + message : message;
```

Comment:
Why don't you want the prefix to be colored ?

Reply:
This was just to match the test case that was there. It asserted that the prefix would not be highlighted even when the rest of the message was, so when it was failing, I changed the code to make the test pass. But I was also not sure that made the most sense. I guess having the actual output highlighted vs the prefix might help draw attention to it, but I'm happy to change this.

Reply:
I'd prefer to keep it like it was (as there was no issue with that yet). So would be awesome if you could fix the test instead (wondering though why it not failed before ...)

Reply:
I restored this.

I have a suspicion that the test wasn't failing because the ANSI logging was being disabled during forked tests since System.console() would return null. Because the tests would use the static Ansi.ansi() method to retrieve a helper to generate the expected output, it would also magically stop putting in the escape sequences, so the expected string would match.

Part of the fix for this was to instead construct a new Ansi() instance to use to construct the expected result so that it would always insert the codes.

While testing this I discovered some other bugs - if the emphasis functionality was used and the emphasis ended at the end of the message, it would fail to reset for the next line. I fixed this.

I also spent a bit of time getting the maven test suite to output the ANSI codes during testing. Because there is static usage going on, the test run individually behaved differently to when it was in the suite. Plus, JAnsi would often disable itself because the output was not a console, so in the end I have forced these codes to output just during the tests so that it is always possible to see they are working and that the console doesn't get messed up.